template/android/app/build.gradle

```diff
@@ -190,7 +192,13 @@ dependencies {
         implementation jscFlavor
     }
 }
-
+configurations.all {
+    resolutionStrategy {
+        // TODO: Line below required for Detox since version 12.5
+        //   It should removed as soon as the project compiles without it
```

Comment:
did this come from the init script or was this a fix we had to make? is there a backing issue on the detox project for it?

Reply:
This comes from an issue on git https://github.com/wix/Detox/issues/1534#issuecomment-512270679

Reply:
👍 can we just put that in the comment so we don't forget about it?